### PR TITLE
Fix #103: Remove the unwrap method while working with current directory

### LINKED ISSUE #103: Remove the unwrap method while working with current directory
Instead of using the [unwrap](https://github.com/mgrachev/dotenv-linter/blob/master/src/lib.rs#L65) method and getting `&str`, we can use the type `&OsStr` without the method `unwrap`. Need to replace `&str` with `&OsStr` in that file.

### PR DESCRIPTION
**Pull request by kanapuli**
closes #103 
current_dir.to_str is replaced with as_os_str which yields the OsString.
The argument type of get_args() function is changed from &str to &OsStr

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -21,7 +22,7 @@ fn get_args(current_dir: &str) -> clap::ArgMatches {
             Arg::with_name("input")
                 .help("files or paths")
                 .index(1)
-                .default_value(current_dir)
+                .default_value(current_dir.to_str().unwrap())
```
Comment (mgrachev) [inline src/lib.rs]:
```suggestion
                .default_value_os(current_dir)
```
It will be better this way

**Comment by mgrachev** [comment]
@kanapuli 👋Thank you for your contribution! 🚀

I have left a comment. Could you fix it and, as well as add this PR to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list)?

File: src/lib.rs (hunk 1 above)
Comment (kanapuli) [reply src/lib.rs]: This is done.

**Comment by mgrachev** [comment]
@kanapuli I see changes in your PR from another contributors. I can't merge such PR.

Can you delete your fork and re-fork the project again and then create another PR only with you changes?

**Comment by kanapuli** [comment]
Raised a new PR https://github.com/mgrachev/dotenv-linter/pull/115. 
Can I close this pull request #109 ?

**Pull request closed without merging**